shell/components/LazyImage.vue

```diff
@@ -17,32 +19,87 @@ export default {
     },
   },
 
+  data() {
+    return {
+      observer:    null,
+      intersected: false,
+      boundError:  null,
```

Comment:
question: Vue will add reactivity to all properties in `data()` - do any of these properties need reactivity attached?

I ask this question because I noticed that `boundError` was previously a non-reactive prop attached to the component instance. I don't think that this PR really changes the way we expect to interact with `boundError`, but I could be missing somthing.. I also wonder if we want `observer` to be reactive as well, it could be needless overhead for this use-case.